projects/valtimo/components/src/lib/components/modal/modal.component.ts

```diff
@@ -24,14 +35,22 @@ declare var $;
   templateUrl: './modal.component.html',
   styleUrls: ['./modal.component.scss'],
 })
-export class ModalComponent implements AfterViewInit {
+export class ModalComponent implements AfterViewInit, OnInit, OnDestroy {
   @Input() elementId: string;
   @Input() title = '';
   @Input() subtitle = '';
   @Input() templateBelowSubtitle: TemplateRef<any>;
   @Input() showFooter = false;
 
-  constructor() {}
+  @ViewChild('scrollModal') scrollModal: ElementRef<any>;
+
+  private scrollSubscription: Subscription;
```

Comment:
`private scrollSubscription!: Subscription;` to make clear it's not initialized with a value